dlc-oracle-test/src/test/scala/org/bitcoins/dlc/oracle/DLCOracleTest.scala

```diff
@@ -0,0 +1,123 @@
+package org.bitcoins.dlc.oracle
+
+import java.sql.SQLException
+
+import org.bitcoins.crypto._
+import org.bitcoins.testkit.BitcoinSTestAppConfig.tmpDir
+import org.bitcoins.testkit.fixtures.BitcoinSFixture
+import org.bitcoins.testkit.util.FileUtil
+import org.scalatest.FutureOutcome
+
+import scala.concurrent.Future
+
+class DLCOracleTest extends BitcoinSFixture {
+
+  override type FixtureParam = DLCOracle
+
+  override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
+    val builder: () => Future[DLCOracle] = () => {
```

Comment:
nit: Maybe move this into testkit? It's nice to have test utilities available to you in other projects